src/tests/commentcount.test.js

```diff
@@ -0,0 +1,19 @@
+import { commentCount } from '../modules/commentsCounter.js';
+/* eslint-disable */
```

Comment:
- [ ] Kindly enable the linters here too. 🤓

Reply:
[issue resolved](https://github.com/Wahaj-Ali/tv-hub/commit/fba0814fbe26e8fa778360170466e12dd48c34ac)